src/askui_runner/modules/core/infrastructure/runner/askui.py

```diff
@@ -49,6 +49,7 @@ def wait_for_controller_to_start(host: str, port: int):
             time.sleep(10)
 
 class AskUiJestRunnerService(Runner):
+    _TEMPLATE_EXTENSION = "jinja"
```

Comment:
Why not
```suggestion
    _TEMPLATE_EXTENSION = ".jinja"
```
?
Should make code simpler and be more in line with the fullstap generally being attributed to the extension.

Reply:
`jinja_env.list_templates` will append a dot to the given extension.
As a result, it will search for files with the extension "..jinja". This behavior caused the template list to be empty, which is why I introduced the exception.